static_frame/core/frame.py

```diff
@@ -187,13 +187,23 @@ def from_concat(cls,
         # axis_series = (0 if axis is 1 else 1)
         frames = [f if isinstance(f, Frame) else f.to_frame(axis) for f in frames]
 
+        own_columns = False
+        own_index = False
+
+        # End quickly if empty iterable
+        if len(frames) == 0:
+            return cls(
+                    index=index,
+                    columns=columns,
+                    name=name,
+                    own_data=True,
```

Comment:
own_data should only be true if the passed data object is an ownable TypeBlocks instance. Here, it may not matter, but it is best left unset.

Reply:
Thanks. I was a bit unsure of what the parameters meant, so I attempted to mimic the original source as closely as possible. It makes more sense to remove it though.